lib/vm/src/vmcontext.rs

```diff
@@ -762,6 +792,17 @@ pub struct VMCallerCheckedAnyfunc {
     // If more elements are added here, remember to add offset_of tests below!
 }
 
+impl VMCallerCheckedAnyfunc {
+    /// Check if this anyfunc is a null funcref.
+    // TODO: we probably want to clean this up before shipping, maybe actually add
+    // an extra layer of indirection
```

Comment:
I did add that layer of indirection, this is a really out of date comment.

Note to self, clean this up